rdmd_test.d

```diff
@@ -310,6 +310,15 @@ void runTests()
         assert(res.status == -SIGSEGV, format("%s", res));
     }
 
+    /* https://issues.dlang.org/show_bug.cgi?id=11997- rdmd should search it's
+    binary path for the compiler */
+    string localDMD = buildPath(tempDir(), baseName(compiler));
+    std.file.write(localDMD, "empty shell");
+    scope(exit) std.file.remove(localDMD);
+    res = execute(rdmdApp ~ [modelSwitch, "--force", "--chatty", "--eval=writeln(`Compiler found.`);"]);
+    assert(res.status == 1, res.output);
+    assert(res.output.canFind(`spawn ["` ~ localDMD ~ `",`));
```

Comment:
Added the test: it would be nice if we didn't expect it to fail, but that would mean copying a whole dmd install over, which would be too much of a PITA.  I'm not sure if the `--force` flag is necessary, didn't need it myself, but all other invocations of `--eval` in this test file use it, so I stuck it here too.